### ISSUE TITLE
<mj-divider> margin problems

### ISSUE THREAD

**Issue by laurent-bientz** [mjmlio/mjml#45]
Hi guyz,

I got a problem with `<mj-divider>`.

I want a separator (_a kind of border in fact_) **with no spaces or margin around**.

Even if I set a margin & padding at 0px on `<mj-divider>` and on `<mj-section>`, the generated HTML code is wrapped inside a `<p>` with 13px margin: 13px 0;

__What i do:__
```html
<mj-body>
  
  <mj-section background-color="grey">
    <mj-column>
      <mj-image width="100" src="/assets/img/logo-small.png"></mj-image>
    </mj-column>
  </mj-section>
  
  <mj-section margin-top="0px" margin-bottom="0" padding-top="0px" padding-left="0px" padding-bottom="0px" padding-right="0px">
  	<mj-divider border-color="#F45E43" padding-top="0px" padding-left="0px" padding-bottom="0px" padding-right="0px"></mj-divider>
  </mj-section>
  
</mj-body>
```

__What I expected:__

![wanted](https://cloud.githubusercontent.com/assets/6093572/12979965/8257548a-d0da-11e5-880b-a18d06f4df00.jpg)

__What I get:__

![result](https://cloud.githubusercontent.com/assets/6093572/12980039/e8158be8-d0da-11e5-82cf-959569930b05.jpg)

Maybe I miss something but i didn't see any CSS reset.

In general, I understand that the tool was initially designed to facilitate the building of newsletter but I regret that we don't get a reset mode.Override each padding of each component is a bit tedious.
What do you think about that?

Thanks

**Comment by iRyusa** [comment]
Hey there,

Sorry, it's fixed in PR #27 and hasn't release yet. 1.0.4 will be push really soon. 

As it's current state, MJML doesn't provide yet a way to reset fully the css. It will come in a near future

**Comment by drazik** [comment]
Hello @iRyusa ,

Do you already have plans for a reset, or we can propose a PR with our own interpretations ?

Thanks.

**Comment by iRyusa** [comment]
@drazik feel free to contribute :+1:  ! I'll have to see if I can give it a shot next week

**Comment by iRyusa** [comment]
Fix in 1.0.4 and 1.1.0 @laurent-bientz :+1:

**Comment by laurent-bientz** [comment]
Thank you @iRyusa !
